wire-test-utils/src/main/java/com/squareup/wire/schema/RepoBuilder.kt

```diff
@@ -40,14 +38,10 @@ import java.nio.file.Files
 class RepoBuilder {
   private val fs = Jimfs.newFileSystem(Configuration.unix())
   private val root = fs.getPath("/source")
-  private val schemaLoader = SchemaLoader().addSource(root)
+  private val schemaLoader = NewSchemaLoader(fs)
 
   fun add(name: String, protoFile: String): RepoBuilder {
-    if (name.endsWith(".proto")) {
-      schemaLoader.addProto(name)
-    } else if (!name.endsWith(".wire")) {
-      throw IllegalArgumentException("unexpected file extension: $name")
-    }
+    check(name.endsWith(".proto") || name.endsWith(".wire")) { "unexpected file extension: $name" }
```

Comment:
```suggestion
    require(name.endsWith(".proto") || name.endsWith(".wire")) { "unexpected file extension: $name" }
```
To keep throwing `IllegalArgumentException`

Reply:
Fixed!